src/components/PartyChip.js

```diff
@@ -4,10 +4,17 @@ import PropTypes from 'prop-types';
 
 function PartyChip({ name, party }) {
   const [display, setDisplay] = useState(true);
+
   function removeParty(e) {
     e.preventDefault();
     const parties = JSON.parse(localStorage.getItem('parties'));
-    delete parties[name];
+    // Grab index of the existing party
+    let index = -1;
```

Comment:
> The findIndex() method returns the index of the first element in the array that satisfies the provided testing function. Otherwise, it returns -1, indicating that no element passed the test.

so you actually don't need this line

Reply:
oof you right